public/scripts/app.js

```diff
@@ -163,19 +256,21 @@ function drawBodyRows(klasses) {
 
 
 		$('tbody').append('<tr id="' + value + '"></tr>').find('#' + value).append(classItem);
-		drawAnchorButton('Edit', '/classes/edit/', 'btn-info', value);
-		drawLbButtons(value, 'danger', 'send-to-del', 'button', 'Delete');
-		drawAnchorButton('Go to Class', '/classes/class/view/', 'btn-default', value);
+		//this will have to change because dropdown menu on small screens
+		drawAnchorButton('Edit', '/classes/edit/', value);
+		drawDelButton(value);
+		drawAnchorButton('Go to Class', '/classes/class/view/', value);
+		drawDropdown(value, i);
 		$('.js-send-to-del-trigger').val(value);
 	}
 }
 
 function drawHeadRow(klasses) {
-	let arr = ['Class Name', 'Subject', 'Grade Level', 'Term'],
+	let arr = ['Class', 'Subject', 'Grade', 'Term'],
 		lng = arr.length;
 
 	for (let i = 0; i < lng; i++) {
```

Comment:
Not a huge difference...but consider map
`arr.map(function(itm){ /*lines 273,274*/ })`